pkg/config/config.go

```diff
@@ -95,6 +96,7 @@ func selectRandom(array []string) string {
 	if len(array) == 0 {
 		return ""
 	}
+	rand.Seed(time.Now().UnixNano())
```

Comment:
seeded once seems sufficient for whole app. suggest to move this rand seeding outside of this function

Reply:
If we do this, even we set up time.Now().UnixNano() as seed, we will get a deterministic sequence of pseudo-random numbers.

Reply:
I added seed as common practice. I also added the link to the golang implementation as a concern that it will offer a deterministic sequence of pseudo-random numbers in this case.